diff --git a/node_modules/playroom/src/Playroom/Frames/frameSrc.js b/node_modules/playroom/src/Playroom/Frames/frameSrc.js
index c08b96d..d195fa5 100644
--- a/node_modules/playroom/src/Playroom/Frames/frameSrc.js
+++ b/node_modules/playroom/src/Playroom/Frames/frameSrc.js
@@ -8,4 +8,4 @@ const defaultFrameSrc = ({ code, themeName }, { baseUrl, paramType }) =>
     code
   )}`;
 
-module.exports = frameConfig.frameSrc ? frameConfig.frameSrc : defaultFrameSrc;
+export default frameConfig.frameSrc ? frameConfig.frameSrc : defaultFrameSrc;
